compiler/src/iree/compiler/Preprocessing/TransformExtensions/PreprocessingExtensions.cpp

```diff
@@ -310,13 +310,23 @@ IREE::transform_dialect::MatchContractionOp::matchOperation(
 DiagnosedSilenceableFailure IREE::transform_dialect::MatchDimsEqualOp::apply(
     transform::TransformRewriter &rewriter,
     transform::TransformResults &results, transform::TransformState &state) {
-  ArrayRef<transform::Param> currentDimSizes =
+  ArrayRef<transform::Param> currentDimAttrs =
       state.getParams(getDimensionSizes());
-  ArrayAttr targetDimensionSizes = getExpectedValues();
+  ArrayRef<int64_t> targetDims = getExpectedValues();
```

Comment:
nit: since you this is called in `expected_values` in tablegen, I'd keep similar variables names in the code here. Then you can have `actualDimAttrs` and `expectedDims`, which is a common pairing in any test code that performs pairwise comparisons.